apps/fhir/bluebutton/views/search.py

```diff
@@ -126,6 +126,12 @@ def get_paging_links(base_url, start_index, page_size, count, replay_parameters)
     out = []
     replay_parameters[SIZE_PARAMETER] = page_size
 
+    replay_parameters[START_PARAMETER] = start_index
+    out.append({
+        'relation': 'self',
+        'url': base_url + '?' + urlencode(replay_parameters)
```

Comment:
Is there a way to ensure this ends up as a valid url, for example so it can handle a case where
`base_url = "http://localhost?existing=param`

Also what is `request.build_absolute_uri('?')` returning?

Reply:
That function returns the full path of the current request, including the method (https), domain (sandbox.bluebutton.cms.gov), and path (/v1/fhir/ExplanationOfBenefit), but not including any query parameters. I don't love it, just because it's a bit obfuscated, but it servers the purpose. What you've suggested above isn't possible, because this will never return any query parameters.